sections/appendix/network.tex

```diff
@@ -0,0 +1,77 @@
+\end{tabular}
+\end{table}
+
+Purpose: The position of a given tag will be continuously transmitted on the multicast channel for all players to receive.
+The positions are sent as values relative to the anchor positions sent during setup, such that a missed package will not impact the future game state.
+The ID (i) \texttt{0} is reserved for the ball tag, and all other tags are considered to be players.
+
+
+\subsection*{1: Send anchor position (TCP)}
+\begin{table}[H]
+\begin{tabular}{|l|l|l|l|}
+\hline
+\textbf{POSY (y)} & \textbf{POSX (x)} & \textbf{ANCHOR\_ID (i)} & \multicolumn{1}{c|}{\textbf{\begin{tabular}[c]{@{}c@{}}TYPE (t)\\ 1\end{tabular}}} \\ \hline
+16 bit            & 16 bit            & 8 bit                   & 8 bit                                                                              \\ \hline
+\end{tabular}
+\end{table}
+
+Purpose: Used as a part of the setup phase, where the host sends the position of one of the four anchors to the player.
+This packet will be sent multiple times to each client, but with various anchor IDs (i) until all clients have received the four positions.
+
+\subsection*{2: Send player tag (TCP)}
+\begin{table}[H]
+\begin{tabular}{|l|l|l|}
+\hline
+\textbf{PLAYER\_ID (p)} & \textbf{TAG\_ID (i)} & \multicolumn{1}{c|}{\textbf{\begin{tabular}[c]{@{}c@{}}TYPE (t)\\ 2\end{tabular}}} \\ \hline
+8 bit                   & 16 bit               & 8 bit                                                                              \\ \hline
+\end{tabular}
+\end{table}
+Purpose: Used as a part of the setup phase, where the host sends the related tag ID to the player, such that they can find the physical tag related to their client instance.
+This packet will be sent once to each client, but with various tag IDs (i) until all clients have received their tag ID.
+
+\subsection*{3: Signal game start (TCP)}
+\begin{table}[H]
+\begin{tabular}{|l|}
+\hline
+\multicolumn{1}{|c|}{\textbf{\begin{tabular}[c]{@{}c@{}}TYPE (t)\\ 3\end{tabular}}} \\ \hline
+8 bit                                                                               \\ \hline
+\end{tabular}
+\end{table}
+
+Purpose: Used to signal to the players that the game has been started.
+
+\subsection*{4: Send goal scored (TCP)}
+\begin{table}[H]
+\begin{tabular}{|l|l|l|}
+\hline
+\textbf{TEAM2SCORE (2)} & \textbf{TEAM1SCORE (1)} & \multicolumn{1}{c|}{\textbf{\begin{tabular}[c]{@{}c@{}}TYPE (t)\\ 4\end{tabular}}} \\ \hline
+8 bit                   & 8 bit                   & 8 bit                                                                              \\ \hline
+\end{tabular}
+\end{table}
+
+Purpose: Used to signal the clients that a goal has been scored, and the scoreboard should be updated on their GUI.
+This triggers the host to generate new goal positions.
+
+\subsection*{5: Send new goal position (TCP)}
```

Comment:
det virker ve legentligt ikke det her ift til det der er skrevet, gør det?
sender den ikke midtpunkter for målene?
Så hvis vi sender en packet til hvert team vil de kun kende placeringen på en målzone, men ikke den anden 🤔